bot/bot.go

```diff
@@ -109,7 +122,7 @@ func (b *Bot) Init(ctx context.Context, rtm *slack.RTM, span *trace.Span) error
 
 	botGroups = nil
 
-	b.logf("Initialized %s with ID: %s\n", b.name, b.id)
+	b.logf("Initialized %s with ID (%q) and msgprefix (%q) \n", b.name, b.id, b.msgprefix)
 	params := slack.PostMessageParameters{AsUser: true}
 	childSpan = initSpan.NewChild("b.AnnouncingStartupFinish")
 	_, _, err = b.slackBotAPI.PostMessageContext(ctx, b.users["dlsniper"], fmt.Sprintf(`Deployed version: %s`, b.version), params)
```

Comment:
Since `dlsniper` isn't going to be involved, can we make it so the bot won't message him on deployment. We should maybe have this write to a channel, though.